editor/plugins/script_text_editor.cpp

```diff
@@ -1779,6 +1797,7 @@ ScriptTextEditor::ScriptTextEditor() {
 	code_editor->set_code_complete_func(_code_complete_scripts, this);
 	code_editor->get_text_edit()->connect_compat("breakpoint_toggled", this, "_breakpoint_toggled");
 	code_editor->get_text_edit()->connect_compat("symbol_lookup", this, "_lookup_symbol");
+	code_editor->get_text_edit()->connect_compat("symbol_validate", this, "_validate_symbol");
```

Comment:
This can now be changed to use `connect("symbol_validate", callable_mp(this, &ScriptTextEditor::_validate_symbol))`, and the callback no longer needs to be bound to ClassDB.